local_postgres_docker_setup.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I think that you can say: "we recommend RDS/etc for production, but include a postgres container for development." That seems entirely reasonable.